ota-plus-web/app/reactapp/src/components/device/queue/MtuListItem.jsx

```diff
@@ -32,9 +32,9 @@ class MultiTargetITem extends Component {
                         {status === "waiting" ?
                             <span>Queued, waiting for device to connect</span>
                         : status === "downloading" ?
-                            <span>Downloading</span>
+                            <span>Waiting for Download to complete</span>

```

Comment:
No need to capitalize `Download`? @EpicNuts @christianhuggerATS

Reply:
@jerrytrieu Sorry, that was my mistake. I should have passed it through Merlin, I guess?

Reply:
@jerrytrieu should I change that?

Reply:
Yes, and `Install` too below.